ksql-parser/src/main/java/io/confluent/ksql/schema/ksql/LogicalSchemas.java

```diff
@@ -118,6 +123,25 @@ private static Type sqlType(final Schema schema) {
       return handler.apply(schema);
     }
 
+    private static PrimitiveType toLogicalType(final Schema schema) {
```

Comment:
My preference is to have the code read about what it does _now_, not what it might do at some point in the future.  It's easy enough to rename a method when we implement some new functionality.